en/simulation/hitl.md

```diff
@@ -113,26 +113,24 @@ Follow the appropriate setup steps for your simulator in the following sections.
 
 > **Note** Make sure *QGroundControl* is not running!
 
-1. Update the environment variables: 
+1. Build PX4 with the Gazebo plugins although we only need the Gazebo part:
```

Comment:
What do you mean by "although we only need the Gazebo part"? Is this useful for people know?

The `DONT_RUN=1` change is good!

Perhaps 
```
1. Build (but do not run) PX4 with Gazebo:
```

Reply:
We build SITL but we only need the build of the gazebo plugins.

Reply:
Not sure that answers my question - who cares? But assuming it is important, how about:

> 1. Build PX4 with Gazebo (in order to to build the Gazebo plugins).

Reply:
Yes that's better. I was just trying not to create confusion. So people might be wondering why SITL needs to be built if we're using HITL.